apps/web/package.json

```diff
@@ -17,7 +17,7 @@
     "sass": "node ./scripts/compile-css.js",
     "watch:rollup": "chokidar \"src/client/**/*.js\" --silent -c \"npm run rollup\"",
     "watch:sass": "chokidar \"src/styles/**/*.scss\" --silent -c \"npm run sass\"",
-    "test": "echo 'TOOD: add test script'",
+    "test": "NODE_ENV=test NODE_OPTIONS=--experimental-vm-modules npx jest",
```

Comment:
Use `cross-env` to set the env vars to make sure it won't crash on Windows or what not.

See above.

Reply:
Done